Formula/seexpr.rb

```diff
@@ -22,47 +22,62 @@ class Seexpr < Formula
   depends_on "doxygen" => :build
   depends_on "libpng"
 
+  uses_from_macos "flex" => :build
+
+  on_linux do
+    depends_on "mesa"
+    depends_on "mesa-glu"
+  end
+
   def install
     mkdir "build" do
-      system "cmake", "..", *std_cmake_args, "-DUSE_PYTHON=FALSE"
+      args = std_cmake_args + %W[
+        -DCMAKE_INSTALL_RPATH=#{rpath}
+        -DUSE_PYTHON=FALSE
+      ]
+      if OS.linux?
+        args << "-DENABLE_LLVM_BACKEND=FALSE"
+        args << "-DENABLE_QT5=FALSE"
```

Comment:
Should we just set these arguments unconditionally?

Reply:
It definitely doesn't currently use `qt@5` on macOS, and I've confirmed it doesn't link to LLVM either.  So we can make these unconditional.